test/Record.purs

```diff
@@ -135,7 +136,7 @@ traverseRecord :: forall f k rin rout.
   f { | rout }
 traverseRecord k =
   map (flip Builder.build {}) <<<
-    hfoldlWithIndex (TraverseProp k) (pure identity)
+    hfoldlWithIndex (TraverseProp k :: TraverseProp f k) (pure identity)
```

Comment:
@monoidmusician noticed that, after updating the functional dependencies, it's also necessary to annotate this now. This makes the library a little less user-friendly in that inference will nicely cover less cases.